### PR TITLE
Add changes for hybrid sync manager and one missed file for AILTN

### PR DESCRIPTION
**Pull request by kchitalia**

### CODE REVIEW

File: SalesforceSDK/Hybrid.SmartSync/Manager/SyncManager.cs [hunk 1]
```diff
@@ -90,30 +90,34 @@ public Models.SyncState GetSyncStatus(long syncId)
...
-            return Models.SyncState.FromJson(syncState);
+            var state = _syncManager.SyncUp(JsonConvert.DeserializeObject<SyncUpTarget>(syncUp), SyncOptions.FromJson(JObject.Parse(syncOptions)), soupName, s =>
+            {
+                new SyncEvent().OnSyncEventRaised(s);
+            });
+            var syncState = JsonConvert.SerializeObject(state);
+            return JsonConvert.DeserializeObject<Models.SyncState>(syncState);
         }
 
         public Models.SyncState ReSync(long syncId, string callback)
```
Comment (helenren) [inline SalesforceSDK/Hybrid.SmartSync/Manager/SyncManager.cs]: Seem you don't use call back for syncdown and syncup, does ReSync need to do the similar change as well?

**Comment by helenren** [comment]
Just one comments/question, others looks good for me. @kchitalia

**Comment by bhariharan** [comment]
👍

File: SalesforceSDK/Hybrid.SmartSync/Manager/SyncManager.cs (hunk 1 above)
Comment (kchitalia) [inline SalesforceSDK/Hybrid.SmartSync/Manager/SyncManager.cs]: @helenren I'll sync up with you offline about your question/comment.

**Pull request merged**